src/backend/distributed/master/master_delete_protocol.c

```diff
@@ -268,12 +269,13 @@ master_drop_sequences(PG_FUNCTION_ARGS)
 	Datum sequenceText = 0;
 	bool isNull = false;
 	StringInfo dropSeqCommand = makeStringInfo();
-	bool coordinator = IsCoordinator();
 
 	CheckCitusVersion(ERROR);
 
-	/* do nothing if DDL propagation is switched off or this is not the coordinator */
-	if (!EnableDDLPropagation || !coordinator)
+	EnsureCoordinator();
```

Comment:
I think this function always gets called when anything is dropped on Postgres, so that's why we don't want an explicit error.

Reply:
Makes sense, I'll undo that and add a test.

Reply:
Should be ready for review again.